Yeast_Project/Scripts/3_Oi1_Output_eval.ipynb

```diff
@@ -0,0 +1,564 @@
+    "df = pd.read_csv(\"Raw_data\\yeast_pcsf_network.sif\", sep = '\\t', names=['Edge'])\n",
+    "\n",
+    "#My Ensemble network output\n",
+    "df_ens = pd.read_csv(\"Required_files\\ensemble-pathway.txt\", sep = '\\t')\n",
+    "\n",
+    "#Gold Standard ID's\n",
+    "gold = pd.read_csv(\"Raw_data\\goldStandardUnionDetailed.txt\", sep = '\\t')\n",
+    "\n",
+    "# #Edge Freq from Yeast study\n",
+    "edge_freq = pd.read_csv(\"Raw_data\\_edgeFreq.eda\", sep = '\\t')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<div>\n",
+       "<style scoped>\n",
+       "    .dataframe tbody tr th:only-of-type {\n",
+       "        vertical-align: middle;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe tbody tr th {\n",
+       "        vertical-align: top;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe thead th {\n",
+       "        text-align: right;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>Edge</th>\n",
+       "      <th>Frequency</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>YJR059W pp YPR074C</td>\n",
+       "      <td>1.000000</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1</th>\n",
+       "      <td>YBR160W pp YDR239C</td>\n",
+       "      <td>1.000000</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2</th>\n",
+       "      <td>YJR059W pp YPR091C</td>\n",
+       "      <td>0.325000</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>3</th>\n",
+       "      <td>YNL166C pp YPL031C</td>\n",
+       "      <td>1.000000</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>4</th>\n",
+       "      <td>YJR059W pp YKL064W</td>\n",
+       "      <td>0.327000</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "                 Edge  Frequency\n",
+       "0  YJR059W pp YPR074C   1.000000\n",
+       "1  YBR160W pp YDR239C   1.000000\n",
+       "2  YJR059W pp YPR091C   0.325000\n",
+       "3  YNL166C pp YPL031C   1.000000\n",
+       "4  YJR059W pp YKL064W   0.327000"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "#Split the freq by = and then concat the freq to the Yeast Osmotic Stress network df\n",
+    "\n",
+    "edge_freq[['Edge', 'Frequency']] = edge_freq[\"EdgeFrequency\"].str.split('=', expand=True)\n",
+    "\n",
+    "edge_freq = edge_freq.drop(columns=['EdgeFrequency','Edge'])\n",
+    "\n",
+    "df = pd.concat([df,edge_freq], axis=1)\n",
```

Comment:
What is this concat doing?

Reply:
The TPS paper edge results file does not have the edge frequencies. Those frequencies came from a different file. I loaded in both files and needed to concat the edge frequencies in order to sort the data based on a desired threshold. Did not end up needing this because my results were better when I kept all of the results.